priceupdater/priceupdater.go

```diff
@@ -331,15 +335,18 @@ func (p *PriceUpdater) UpdateTokenList() error {
 			}
 		}
 		for _, provider := range p.providers {
-			if len(provider.SymbolsMap.Symbols) != 0 {
+			switch provider.Provider {
+			case UpdateMethodTypeBitFinexV2:
 				if _, ok := provider.SymbolsMap.Symbols[dbToken.TokenID]; !ok {
 					provider.SymbolsMap.Symbols[dbToken.TokenID] = dbToken.Symbol
 				}
-			}
-			if len(provider.AddressesMap.Addresses) != 0 {
+			case UpdateMethodTypeCoingeckoV3:
 				if _, ok := provider.AddressesMap.Addresses[dbToken.TokenID]; !ok {
 					provider.AddressesMap.Addresses[dbToken.TokenID] = dbToken.Addr
 				}
+			default:
+				log.Error("Unknown provider detected: ", provider.Provider)
```

Comment:
Here I would suggest to log an info instead of error. Why? Because maybe the hermez node administrator intentionally  try to change to another price service and would be good to us to inform her that we don't support it. 
```log.Warn("This price service provider is not supported: ", provider.Provider)```